schemas/input/csl-data.json

```diff
@@ -256,6 +256,10 @@
               "description": "E.g. rendered as 'Smith, J. [@JSmith]'",
               "type": "string"
             },
+            "label": {
+              "title": "Non-default label to use when rendering name",
+              "description": "Overrides the default label for the role. e.g. to indicate a subtype of a role or specific MARC or CRediT roles."
```

Comment:
Also, don't we need a `"type": "string"` here?

Reply:
Also, not sure how this happened (seemed to happen with the last merge I did), but there are multiples places where the $ref syntax got screwed up. We need to add to the python test code to catch these problems in the future.

You can fix by find-replace `#definition` -> `#/definition`.

Reply:
> Yes this being locale-specific may be an issue. But I assume @bwiernik has already an idea how this could play nicely with #240

He mentions this in [this comment](https://github.com/citation-style-language/schema/issues/293#issuecomment-655500060), which I missed earlier.

How does this PR relate to the idea in that comment @bwiernik?

Is this really an uncontrolled term name?